Assets/MixedRealityToolkit/_Core/Devices/BaseController.cs

```diff
@@ -23,14 +23,39 @@ public abstract class BaseController : IMixedRealityController
         /// <param name="controllerHandedness"></param>
         /// <param name="inputSource"></param>
         /// <param name="interactions"></param>
-        protected BaseController(TrackingState trackingState, Handedness controllerHandedness, IMixedRealityInputSource inputSource = null, MixedRealityInteractionMapping[] interactions = null)
+        protected BaseController(TrackingState trackingState, Handedness controllerHandedness, IMixedRealityInputSource inputSource, MixedRealityInteractionMapping[] interactions)
```

Comment:
please revert, the nulls are there for a valid reason.

Controllers don't have a dependency on the input system and need to be able to operate without an input source or mappings.

Reply:
This was done as part of adding the other constructors.